lib/child_process.js

```diff
@@ -242,10 +244,11 @@ exports.execFile = function(file /*, args, options, callback*/) {
 
   if (child.stdout) {
     child.stdout.addListener('data', function(chunk) {
-      stdoutLen += chunk.byteLength;
+      stdoutLen += (chunk.byteLength || chunk.length);
```

Comment:
Wait, what? Doesn't this bring us back to the original problem if `chunk` is a string?